handler_templates.ts

```diff
@@ -354,7 +354,5 @@ export function generateObjectResponseHandler<T extends Schema>(
     if (!finalSchema) {
       return projectedBody;
     }
-
-    return transformBody(projectedBody, finalSchema);
```

Comment:
do you just want to return `projectedBody` regardless of `finalSchema` here?

Reply:
Yes. Done.